packages/cw1155/README.md

```diff
@@ -54,10 +54,10 @@ Any contract wish to receive CW1155 tokens must implement `Cw1155ReceiveMsg` and
 
   `from`/`to` are optional, no `from` attribute means minting, no `to` attribute means burning.
 
-- `metadata(url, token_id)`
+- `token_info(url, token_id)`
```

Comment:
In event.rs i only see MetadataEvent, not found token_info.
And i dont see any api using this event.
I just want add uri for token, but dont find any api do this?

Reply:
metadata modification is not part of the standard API, different contracts can implement their own APIs to do that I think.

Reply:
Do you mean base on ERC1155Metadata?
Do you think it can fulfill what's missing?